community/kernel/src/test/java/org/neo4j/kernel/impl/newapi/SimpleNodeValueClient.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2002-2017 "Neo Technology,"
+ * Network Engine for Objects in Lund AB [http://neotechnology.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.neo4j.kernel.impl.newapi;
+
+import org.neo4j.storageengine.api.schema.IndexProgressor;
+import org.neo4j.values.storable.Value;
+
+public class SimpleNodeValueClient implements IndexProgressor.NodeValueClient
```

Comment:
As an alternative, this could live together with `IndexProgressor` in `org.neo4j.storageengine.api.schema`

Reply:
I don't see any use for this class outside of tests, so let's keep it out of production code.

Reply:
Yes... I'm suggesting to put it in `community/kernel/src/test/org/neo4j/storageengine/api/schema`?

Reply:
Haha, alright. Sounds good!